crates/neon/src/types_impl/mod.rs

```diff
@@ -182,12 +243,38 @@ impl private::ValueInternal for JsUndefined {
     }
 }
 
-/// The JavaScript `null` value.
+/// The type of JavaScript
+/// [`null`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#primitive_values)
+/// primitives.
+///
+/// # Example
+///
+/// ```
+/// # use neon::prelude::*;
+/// # fn test(mut cx: FunctionContext) -> JsResult<JsNull> {
+/// // Extract the console.log function:
+/// let console: Handle<JsObject> = cx.global().get(&mut cx, "console")?;
+/// let log: Handle<JsFunction> = console.get(&mut cx, "log")?;
+///
+/// // The null value:
+/// let null = cx.null();
+///
+/// // Call console.log(undefined):
+/// log.call_with(&cx).arg(null).exec(&mut cx)?;
```

Comment:
Similar advice as above. There might be value in having multiple lines for doc clarity, but this example _could_ be a one-liner:

```suggestion
/// cx.global()
///     .get::<JsObject, _, _>(&mut cx, "console")?
///     .call_method_with(&mut cx, "log")?
///     .arg(cx.null())
///     .exec(&mut cx)?;
```

Reply:
love it!